test/ktn_meta_suite_SUITE.erl

```diff
@@ -14,9 +15,29 @@
 -type config() :: [{atom(), term()}].
 
 -spec init_per_suite(config()) -> config().
+-ifdef(rebar).
+init_per_suite(Config) ->
+    LibDir = code:lib_dir(katana),
+    BuildRoot = filename:dirname(filename:dirname(LibDir)),
+    {ok, Dirlist} = file:list_dir(BuildRoot),
+    [Plt|_] = lists:filter(fun (File) ->
```

Comment:
Since `plts` is a list, you can simply use the whole list instead of picking the first one. That way, if you have no plts there the error will be more descriptive (it will be the error that `ktn_meta_SUITE` generates when plts are missing).
As a matter of fact, I think you could've just used…
```erlang
Plts = [filename:join(BuildRoot, Plt) || Plt <- filelib:wildcard("*.plt", BuildRoot)],
```